tox.ini

```diff
@@ -42,6 +43,7 @@ deps=
 	django41: Django ~= 4.1
 	django42: Django ~= 4.2
 	django50: Django ~= 5.0
+	django51: Django ~= 5.1a1
```

Comment:
Shouldn’t this be `5.1` instead of `5.1a1` ?

Reply:
😁
![CleanShot 2024-06-26 at 19 09 34@2x](https://github.com/celery/django-celery-beat/assets/4662342/07bcda6e-9d7e-40c1-bb07-09df508eb46a)

Reply:
```suggestion
	django51: Django ~= 5.1b1
```